Source/player.cpp

```diff
@@ -1791,16 +1786,13 @@ StartPlayerKill(int pnum, int earflag)
 						ear._ivalue = player._pLevel;
 
 						if (FindGetItem(IDI_EAR, ear._iCreateInfo, ear._iSeed) == -1) {
-							PlrDeadItem(player, &ear, 0, 0);
+							PlrDeadItem(player, &ear, { 0, 0 });
 						}
 					} else {
-						pi = &player.InvBody[0];
-						i = NUM_INVLOC;
-						while (i--) {
-							pdd = (i + player._pdir) & 7;
-							Point offset = Point { 0, 0 } + static_cast<Direction>(pdd);
-							PlrDeadItem(player, pi, offset.x, offset.y);
-							pi++;
+						Direction pdd = player._pdir;
+						for (int i = 0; i < NUM_INVLOC; i++) {
+							pdd = left[pdd];
+							PlrDeadItem(player, &player.InvBody[i], Point::fromDirection(pdd));
 						}

```

Comment:
There is a slight change to the logic here, the original would always drop the last item at position `-left[player._pdir]`, instead the new code will always drop the first item at position `+left[player._pdir]`.
I don't think the difference warrants keeping the extra logic as it not really relevant to anything.